### DISCUSSION TITLE
Show the borders of the areas by default and just change the border color on hover

### DISCUSSION THREAD

**Discussion by Loligplayer33** [img-mapper/img-mapper#49] [Q&A] [answered]
Essentially what I am looking for is something like the "preFillColor" property, just not for the whole are area but just the borders so that one can identify where the areas are without having to hover over the picture.  Is there any way to achieve this?

Thank you very much in advance!

**Comment by NishargShah** [comment, marked as answer]
Yes, it is possible

Just add `transparent` in your `preFillColor`.

![image](https://user-images.githubusercontent.com/35843261/196033853-f512bc88-2bb8-4dd7-9b62-ca17d607c9ba.png)

CodeSandbox: https://codesandbox.io/s/affectionate-sun-ud0qib
